File: semester I/Praktikum Pemrograman I/Modul6_JovanGilbertNatamasindah_2310817310002/Python/PRAK605-2310817310002-JovanGilbertNatamasindah.py
import numpy as np

ORDO = int(input("Masukkan jumlah ordo matriks : "))

print("Matriks A:")
ELEMEN_ARRAY1 = []
for i in range(ORDO):
    BARIS = list(map(int, input("").split()))
    ELEMEN_ARRAY1.append(BARIS)

print("Matriks B:")
ELEMEN_ARRAY2 = []
for i in range(ORDO):
    BARIS = list(map(int, input("").split()))
    ELEMEN_ARRAY2.append(BARIS)

arr1 = np.array(ELEMEN_ARRAY1).reshape(ORDO,ORDO)
arr2 = np.array(ELEMEN_ARRAY2).reshape(ORDO,ORDO)

print("Matriks AXB : ")

PERKALIAN = np.dot(arr1, arr2)

for HASIL in PERKALIAN :
    print(*HASIL) 


# mengubah disini untuk mengetes commit melalui vscode ke github

File: semester I/Praktikum Pemrograman I/Modul6_JovanGilbertNatamasindah_2310817310002/C/PRAK605-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

void main() {
    int ORDO,BARIS1,BARIS2;

    printf("Masukkan jumlah ordo matriks : ");
    scanf("%d", &ORDO);

    int arrA[ORDO][ORDO];
    int arrB[ORDO][ORDO];

    printf("Matriks A : \n");
    for (int i = 0 ; i < ORDO ; i++) {
        for (int j = 0 ; j < ORDO ; j++) {
            scanf("%d", &arrA[i][j]);
        }
    }

    printf("Matriks B : \n");
    for (int i = 0 ; i < ORDO ; i++) {
        for(int j = 0 ; j < ORDO ; j++) {
            scanf("%d", &arrB[i][j]);
        }
    }

    int HASIL[ORDO][ORDO];

    for (int i = 0; i < ORDO; i++) {
        for (int j = 0; j < ORDO; j++) {
            HASIL[i][j] = 0;
            for (int h = 0; h < ORDO; h++) {
                HASIL[i][j] += arrA[i][h] * arrB[h][j];
            }
        }
    }

    printf("Matriks AXB : \n");
    for (int i = 0; i < ORDO; i++) {
        for (int j = 0; j < ORDO; j++) {
            printf("%d ", HASIL[i][j]);
        }
        printf("\n");
    }
}